alibuild_helpers/build_template.sh

```diff
@@ -80,8 +80,8 @@ fi
 mkdir -p "$INSTALLROOT" "$BUILDROOT" "$BUILDDIR" "$WORK_DIR/INSTALLROOT/$PKGHASH/$PKGPATH"
 
 cd "$WORK_DIR/INSTALLROOT/$PKGHASH"
-cat <<\EOF | tr \' \" >"$INSTALLROOT/.full-dependencies"
```

Comment:
I think we do use `.full-dependencies` in a few places. I would simply add a new file.

Reply:
Do you know where? I can't find it being used anywhere in alibuild, ali-bot or alidist.

Reply:
Hi @awegrzyn, do you use the `.full-dependencies` file, e.g. in your RPM publisher?

Reply:
No, we do not use it atm, although if it also included build dependencies it would very useful for us.

Reply:
OK, I suggest we remove it (as discussed, I think at some point we were using it), and add everything in the new file you have in this PR. Please make sure the full dependencies are there as well, so that @awegrzyn is happy.

Reply:
Done. @awegrzyn: the .meta.json file will have a `dependencies` key, which should include all the info you need (dependencies broken up by build/runtime and whether they're direct deps or transitive deps).